### PR TITLE
Conversation system

### PR DESCRIPTION
**Pull request by misternebula**

### CODE REVIEW

File: QSB/ConversationSync/QSBCharacterAnimController.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using QSB.WorldSync;
+
+namespace QSB.ConversationSync
+{
+    public class QSBCharacterAnimController : WorldObject
```
Comment (amazingalek) [inline QSB/ConversationSync/QSBCharacterAnimController.cs:5]: Unused? :)

File: QSB/PlayerInfo.cs [hunk 2]
```diff
@@ -11,15 +11,19 @@ public class PlayerInfo
...
         public QSBProbe Probe { get; set; }
         public QSBFlashlight FlashLight => Camera?.GetComponentInChildren<QSBFlashlight>();
         public QSBTool Signalscope => GetToolByType(ToolType.Signalscope);
         public QSBTool Translator => GetToolByType(ToolType.Translator);
         public QSBTool ProbeLauncher => GetToolByType(ToolType.ProbeLauncher);
+
         public PlayerHUDMarker HudMarker { get; set; }
         public string Name { get; set; }
         public bool IsReady { get; set; }
+        public int ConversationId { get; set; }
```
Comment (amazingalek) [inline QSB/PlayerInfo.cs:26]: what's the intent? :)

File: QSB/QSB.csproj.user [hunk 3]
```diff
@@ -3,5 +3,6 @@
   <PropertyGroup>
     <GameDir>D:\EpicGames\OuterWilds</GameDir>
     <OwmlDir>C:\Users\Henry\AppData\Roaming\OuterWildsModManager\OWML</OwmlDir>
+    <ProjectView>ShowAllFiles</ProjectView>
```
Comment (amazingalek) [inline QSB/QSB.csproj.user:6]: 🤔

Comment (misternebula) [reply QSB/QSB.csproj.user:6]: was useful for me to find files VS had hidden by accident. like there was a copy of QuaternionHelper in the TransformSync namespace...

**Pull request merged**